### ISSUE TITLE
How to run the unity demo?

### ISSUE THREAD

**Issue by godzil21** [Martin20150405/HelloCV#1]
Hi Martin!

Your work is awesome! , but I was trying to test the unity demo and I'm keeping an error message saying library LibBRA_SLAM could not be loaded.

I believe this is due to dealing with a 32 bits library and I'm using a 64 windows and 64 unity 5 editor. 
Could you give me a hint?  Must I use a windows 32 for testing the demo?

Thanks a lot in advance.

Kind Regards!

**Comment by Martin20150405** [comment]
> Must I use a windows 32 for testing the demo

No, but a x86 editor is recomended. (It's only a half done project.)
Or you can compile a x64 version on your own. Check the Cmakelists in Windows project for more details.
